pages/api/request/help.ts

```diff
@@ -0,0 +1,88 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { cors, middleware, sessionExists } from "@lib/middleware";
+import { NotifyClient } from "notifications-node-client";
+import { logMessage } from "@lib/logger";
+import { MiddlewareProps } from "@lib/types";
+
+const SUPPORT_EMAIL_ADDRESS = "assistance+forms@cds-snc.freshdesk.com";
+const CONTACTUS_EMAIL_ADDRESS = "jose.jimenez@cds-snc.ca";
+
+const requestSupport = async (
+  req: NextApiRequest,
+  res: NextApiResponse,
+  { session }: MiddlewareProps
+) => {
+  try {
+    if (!session) return res.status(403).json({});
+
+    const { type, name, email, request, description } = req.body;
+
+    if (!name || !email || !request || !description) {
+      return res.status(404).json({ error: "Malformed request" });
+    }
+
+    const templateID = process.env.TEMPLATE_ID;
+    const notifyClient = new NotifyClient(
```

Comment:
Perhaps for a follow-up PR but would be nice if this was a utility function to reduce the duplicate code.